packages/manager/modules/pci/src/components/pci-order-command/pci-order-command.scss

```diff
@@ -1,5 +1,16 @@
-.overloader_height_tail_logs {
-  .tail-logs:first-child {
-    height: auto;
+pci-order-command {
+  .overloader_height_tail_logs {
+    .tail-logs:first-child {
+      height: auto;
+    }
+  }
+  .command {
+    background-color: #122844;
```

Comment:
Didi you try to locate if we alredy create a variable for this ?
If it doesn't existe so we use a non valid color regarding our UI chart.

Reply:
This color is used in the `tail-logs` component that was used in `pci-order-command` before. We want to have the same look, but unfortunately, it seems that this color is not exposed in the ui-kit: https://ovh.github.io/ovh-ui-kit/?path=/story/design-system-helpers-backgrounds--primary